tools/quake3/q3map2/q3map2.h

```diff
@@ -2030,9 +2030,9 @@ Q_EXTERN bool nocmdline Q_ASSIGN( false );
 
 Q_EXTERN int patchSubdivisions Q_ASSIGN( 8 );                       /* ydnar: -patchmeta subdivisions */
 
-Q_EXTERN int maxLMSurfaceVerts Q_ASSIGN( 64 );                      /* ydnar */
-Q_EXTERN int maxSurfaceVerts Q_ASSIGN( 999 );                       /* ydnar */
-Q_EXTERN int maxSurfaceIndexes Q_ASSIGN( 6000 );                    /* ydnar */
+Q_EXTERN unsigned int maxLMSurfaceVerts Q_ASSIGN( 64 );             /* ydnar */
+Q_EXTERN unsigned int maxSurfaceVerts Q_ASSIGN( 999 );              /* ydnar */
+Q_EXTERN unsigned int maxSurfaceIndexes Q_ASSIGN( 6000 );           /* ydnar */
```

Comment:
What are profits of turning these to unsigned? In q3map2 most of size variables are signed integers, so this does not complies the rest of code seamlessly, e.g. adds warnings 
![image](https://user-images.githubusercontent.com/1734117/120699507-22657380-c4b9-11eb-81ee-7441c3d90137.png)
May be dangerful too, if user enters negative values, as these are parsed by *atoi()*.
I think these changes ought to happen during global refactoring of size variables types, based on well evaluated principles.

Reply:
Well, the reason for changing them to unsigned is so they can store the 65k value. I'm asuming those ints are shorts. I didn't check that, tbh.

Reply:
These are normal ints, so data format overflow is pretty far to reach.

Reply:
Ah, no problem there then. I'll revert that part